### PR TITLE
ISSUE-183: Patch Facets/Views Blocks and Update composer.json

### PR DESCRIPTION
**Pull request by DiegoPino**
See #183 and https://github.com/esmero/archipelago-deployment-live/issues/50

This adds:
- Modded Patch to allow Facet/Sorts/Filter preservation on Exposed Views Forms (blocks)
- Removes some dependencies
- Adds Plugin allowance and Configure our packages to be fetched from source

This is not the final/cleaned up composer.json and the lock will be matched once we go into release mode

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -120,7 +117,18 @@
...
+            "composer/installers": true,
+            "cweagans/composer-patches": true,
+            "drupal/core-composer-scaffold": true,
+            "drupal/core-project-message": true,
+            "dealerdirect/phpcodesniffer-composer-installer": true
+        },
+        "preferred-install": {
+            "archipelago/*": "source",
+            "strawberryfield/*": "source"
+            "*": "dist"
```
Comment (aksm) [inline composer.json]:
```suggestion
            "strawberryfield/*": "source",
            "*": "dist"
```

Comment (aksm) [reply composer.json]: Just a missing comma.

**Review by aksm — COMMENTED** [review]
Getting the following error when I deploy:

```
Could not apply patch! Skipping. The error was: Cannot apply patch patches/3032353-25-archipelago-05162022.patch
```

**Comment by DiegoPino** [comment]
> Getting the following error when I deploy:
> 
> ```
> Could not apply patch! Skipping. The error was: Cannot apply patch patches/3032353-25-archipelago-05162022.patch
> ```

Hi, you testing with Drupal 9.3?

File: composer.json (hunk 1 above)
Comment (DiegoPino) [reply composer.json]: Thanks! fixed. Let me know if you see anything else.

**Comment by DiegoPino** [comment]
This was tested (the patch) with `drupal/core:9.3.11`. This pull does not include yet an updated composer.lock bc there is more work to do on dependencies and versions for 1.0.0

That said, maybe I got the patch wrong?

**Comment by aksm** [comment]
> This was tested (the patch) with `drupal/core:9.3.11`. This pull does not include yet an updated composer.lock bc there is more work to do on dependencies and versions for 1.0.0
> 
> That said, maybe I got the patch wrong?

My fault! I registered but didn't fully heed your warning about the lock file.

**Review by aksm — APPROVED** [review]

**Comment by DiegoPino** [comment]
Thanks @aksm !

**Pull request merged**